modules/setting/service.go

```diff
@@ -75,6 +75,16 @@ var Service struct {
 	} `ini:"service.explore"`
 }
```

Comment:
Why not change this to:

```suggestion
} {
  AllowedUserVisibilityModesSlice: []bool{true, true, true},
}
```

Reply:
Then you could drop all of those changes in the tests